docs/using/messaging.md

```diff
@@ -11,60 +11,57 @@ title: Messaging (:talk)
 
 <div class="col-md-8">
 
-`:talk` is the Urbit messaging and notifications protocol. Today we use
-`:talk` just to chat and coordinate, but it's really a general purpose
-piece of infrastructure.
+`:talk` is the built-in frontend for the Urbit messaging and notifications
+protocol, `:hall`. Today we use `:hall` just to chat and coordinate, but
+it's really a general purpose piece of infrastructure.
 
 For the time being come join us in `/urbit-meta` by using the
 <a href="#-quickstart">quickstart</a> below.
 
-Today `:talk` is sort of like a distributed, encrypted Slack that can be
-used from the CLI and the browser. There’s no central `:talk` server.
+Today `:hall` is sort of like a distributed, encrypted Slack that can be
+used from the CLI and the browser. There’s no central `:hall` server.
 Any Urbit can host one.
 
-`:talk` is a general purpose tool for both aggregating and publishing
-streams of messages. Applications can use `:talk` as their transport
-protcol, API connectors can push disparate data sources into `:talk`,
+`:hall` is a general purpose tool for both aggregating and publishing
+streams of messages. Applications can use `:hall` as their transport
+protocol, API connectors can push disparate data sources into `:hall`,
 and so on. There are lots of things a distributed message protocol can
 be used for that we haven't even thought of.
 
+Here we'll be discussing how to operate the default CLI frontend, `:talk`,
+to send and receive messages. For a more in-depth look at `:hall`'s
+internals, take a look at its [documentation](#TODO).
+
 </div>
 
 </div>
 
 ## Quickstart
 
 For the most part we use `:talk` as a single-instance of Slack: one main
-channel (`/urbit-meta`) and dms. Everyone is more than welcome in
-`/urbit-meta`. It's the place to get help, ask questions and chat about
+channel (`/urbit-meta`) and direct messages. Everyone is more than welcome
+in `/urbit-meta`. It's the place to get help, ask questions and chat about
 Urbit in general.
 
 Let's join `/urbit-meta`:
 
 Use `ctrl-x` to switch from `:dojo` to `:talk`.
 
-If you're a planet:
-
-    ~your-urbit:talk> ;join /urbit-meta
-
-If you're a comet:
```

Comment:
I'm assuming `;join /urbit-meta` works for comets now on new Talk?

Reply:
We're going to want to host urbit-meta on ~zod (for federation), so yeah.